scheduler/wrapper.go

```diff
@@ -155,6 +155,15 @@ func doStaticHostUpdate(d distro.Distro) ([]string, error) {
 			NeedsReprovision: provisionChange,
 			Provisioned:      provisioned,
 		}
+		if dbHost == nil || dbHost.Status == evergreen.HostTerminated {
```

Comment:
could an alternative be to remove a host from the db when it's removed from its distro?

Reply:
I think that's weird if the host is running a task?

Reply:
Bad things tend to happen when we remove hosts, so I think it's simpler to set it to terminated.